utils.sh

```diff
@@ -137,7 +137,7 @@ function create_cluster() {
     fi
 
     if [ ! -z "${ASSETS_EXTRA_FOLDER:-}" ]; then
-      cp -rf ${ASSETS_EXTRA_FOLDER}/*.yaml ${assets_dir}/openshift/
+      find "${ASSETS_EXTRA_FOLDER}" \( -name \*.yml -or -name \*.yaml \) -exec cp {} "./${assets_dir}/openshift" \;
```

Comment:
I'm not sure if anybody actually sets OCP_DIR but if there do and its an absolute directory is will find its way to $assets_dir , which wont work here with the "./" prefixed

Reply:
Makes sense, fixed.  Not sure why I added it to begin with.